src/pages/ja/bubblegum-v2/collections.md

```
@@ -0,0 +1,49 @@
+const metadata: MetadataArgsV2Args = {
+  ...assetWithProof.metadata,
+  collection: collection?.key ?? null,
+};
+
+const signature = await setCollectionV2(umi, {
+  ...assetWithProof,
+  newCollectionAuthority: newCollectionUpdateAuthority,
+  metadata,
+  newCoreCollection: newCoreCollection.publicKey,
+}).sendAndConfirm(umi);
+```
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)